### ISSUE TITLE
 `npm run protractor` doesn't work on win7 x64

### ISSUE THREAD

**Issue by p270228163** [angular/angular-phonecat#243]
npm-debug.log  file
0 info it worked if it ends with ok
1 verbose cli [ 'D:\\Program Files\\nodejs\\\\node.exe',
1 verbose cli   'D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'protractor' ]
2 info using npm@2.5.1
3 info using node@v0.10.27
4 verbose node symlink D:\Program Files\nodejs\\node.exe
5 verbose run-script [ 'preprotractor', 'protractor', 'postprotractor' ]
6 info preprotractor angular-phonecat@0.0.0
7 verbose unsafe-perm in lifecycle true
8 info protractor angular-phonecat@0.0.0
9 verbose unsafe-perm in lifecycle true
10 info angular-phonecat@0.0.0 Failed to exec protractor script
11 verbose stack Error: angular-phonecat@0.0.0 protractor: `protractor test/protractor-conf.js`
11 verbose stack Exit status 1
11 verbose stack     at EventEmitter.<anonymous> (D:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:213:16)
11 verbose stack     at EventEmitter.emit (events.js:98:17)
11 verbose stack     at ChildProcess.<anonymous> (D:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:14:12)
11 verbose stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
11 verbose stack     at maybeClose (child_process.js:753:16)
11 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:820:5)
12 verbose pkgid angular-phonecat@0.0.0
13 verbose cwd F:\2.grunt\angular-phonecat
14 error Windows_NT 6.1.7601
15 error argv "D:\\Program Files\\nodejs\\\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "protractor"
16 error node v0.10.27
17 error npm  v2.5.1
18 error code ELIFECYCLE
19 error angular-phonecat@0.0.0 protractor: `protractor test/protractor-conf.js`
19 error Exit status 1
20 error Failed at the angular-phonecat@0.0.0 protractor script 'protractor test/protractor-conf.js'.
20 error This is most likely a problem with the angular-phonecat package,
20 error not with npm itself.
20 error Tell the author that this fails on your system:
20 error     protractor test/protractor-conf.js
20 error You can get their info via:
20 error     npm owner ls angular-phonecat
20 error There is likely additional logging output above.
21 verbose exit [ 1, true ]

**Comment by dongidea** [comment]
You should set "chromeOnly" to "false" in protractor-conf.js file then try to run protractor again.
Here is my config file

```javascript
exports.config = {
  allScriptsTimeout: 11000,

  specs: [
    'e2e/*.js'
  ],

  capabilities: {
    'browserName': 'chrome'
  },

  chromeOnly: false,

  baseUrl: 'http://localhost:8000/',

  framework: 'jasmine',

  jasmineNodeOpts: {
    showColors: true,
    defaultTimeoutInterval: 30000
  }
};

```

Hope this helps!

**Comment by imsma** [comment]
the solution  mentioned by dongidea doesn't work on my system (Windows 7 64bit), here are details 

protractor-conf.js
============================================================
exports.config = {
  allScriptsTimeout: 11000,

  specs: [
    'e2e/*.js'
  ],

  capabilities: {
    'browserName': 'chrome'
  },

  chromeOnly: false,

  baseUrl: 'http://localhost:8000/',

  framework: 'jasmine',

  jasmineNodeOpts: {
    defaultTimeoutInterval: 30000
  }
};
============================================================
npm-debug.log

============================================================
0 info it worked if it ends with ok
1 verbose cli [ 'c:\\nodejs\\node.exe',
1 verbose cli   'c:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'protractor' ]
2 info using npm@2.7.4
3 info using node@v0.12.2
4 verbose run-script [ 'preprotractor', 'protractor', 'postprotractor' ]
5 info preprotractor angular-phonecat@0.0.0
6 verbose unsafe-perm in lifecycle true
7 info protractor angular-phonecat@0.0.0
8 verbose unsafe-perm in lifecycle true
9 info angular-phonecat@0.0.0 Failed to exec protractor script
10 verbose stack Error: angular-phonecat@0.0.0 protractor: `protractor test/protractor-conf.js`
10 verbose stack Exit status 1
10 verbose stack     at EventEmitter.<anonymous> (c:\nodejs\node_modules\npm\lib\utils\lifecycle.js:213:16)
10 verbose stack     at EventEmitter.emit (events.js:110:17)
10 verbose stack     at ChildProcess.<anonymous> (c:\nodejs\node_modules\npm\lib\utils\spawn.js:14:12)
10 verbose stack     at ChildProcess.emit (events.js:110:17)
10 verbose stack     at maybeClose (child_process.js:1015:16)
10 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
11 verbose pkgid angular-phonecat@0.0.0
12 verbose cwd d:\sandbox\angular\angular-phonecat
13 error Windows_NT 6.1.7601
14 error argv "c:\\nodejs\\node.exe" "c:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "protractor"
15 error node v0.12.2
16 error npm  v2.7.4
17 error code ELIFECYCLE
18 error angular-phonecat@0.0.0 protractor: `protractor test/protractor-conf.js`
18 error Exit status 1
19 error Failed at the angular-phonecat@0.0.0 protractor script 'protractor test/protractor-conf.js'.
19 error This is most likely a problem with the angular-phonecat package,
19 error not with npm itself.
19 error Tell the author that this fails on your system:
19 error     protractor test/protractor-conf.js
19 error You can get their info via:
19 error     npm owner ls angular-phonecat
19 error There is likely additional logging output above.
20 verbose exit [ 1, true ]
============================================================

**Comment by Isaddo** [comment]
Did you have the error message `RangeError: port should be >= 0 and < 65536: 104125`
I encountered a similar problem, please refer the following post
http://stackoverflow.com/questions/30134415/protractor-on-64-bit-node-have-rangeerror-the-port-excced-the-range-0-65536

**Comment by cathenry** [comment]
I'm getting exactly the same output as the same as imsma above. I tried changing 'chomeOnly' to 'false', with no change. Also Win7 64bit.

**Comment by EliBKNY** [comment]
Thanks dongidea, changing the "ChromeOnly" to false worked! Protractor is now running fine...

**Comment by RyanWittibschlager** [comment]
I came here because I had the same error description as above. With further investigation, I discovered that I had an outdated chromedriver. An issue has already been opened up for this, with a solution:
https://github.com/angular/angular-phonecat/issues/252

If you downloaded the phonecat tutorial between 2014-10-07 and now, you will need to complete this step before being able to run Protractor.
